### ISSUE TITLE
Expose programmatic zoom and pan APIs

### ISSUE THREAD

**Issue by madbolter** [chartjs/chartjs-plugin-zoom#82]
resetZoom is the only exposed method, but we NEED a complete API, including such things as: wheel "click" zoom increment (for large datasets) and zoomIn zoomOut, so that we can in code set up our own buttons to control zooming based upon our recognition of dataset size. The reset method is on the right track but is woefully inadequate by itself!

We also need properties! This plugin is seriously almost useless because of how "closed" it is. For example, we have no way to display to a user what the zoom amount IS. We should be able to populate, for example, a label to show "200%," "322%," "685%," and so on, with a click on a "Reset" button restoring the "100%" zoom and displaying that fact to the user.

In short, the plugin has promise, but at present it's far, far too limited and limiting to be practically useful in any production app.

API DOCUMENTATION!

**Comment by etimberg** [comment]
@madbolter we're happy to look at PRs adding any of the features you mentioned.
